Add Changelog for May 09, 2025 (#4096)
timescale/docs: about/changelog.md

All the latest features and updates to Timescale products.

> ## 🤖 Automatic document embeddings from S3 and Sample dataset for AI testing

Reviewer: ```suggestion
## 🤖 Automatic document embeddings from S3 and a sample dataset for AI testing
```

<Label type="date">May 09, 2025</Label>

### Automatic document embeddings from S3

> Timescale’s  Vectorizer now supports automatic document vectorization, making it dramatically easier to build RAG and semantic search applications on top of unstructured data stored in Amazon S3. With just a SQL command, developers can create, update, and synchronize vector embeddings from a wide range of document formats—including PDFs, DOCX, XLSX, HTML, and more—without building or maintaining complex ETL pipelines.

Reviewer: ```suggestion
pgai vectorizer now supports automatic document vectorization. This makes it dramatically easier to build RAG and semantic search applications on top of unstructured data stored in Amazon S3. With just a SQL command, developers can create, update, and synchronize vector embeddings from a wide range of document formats—including PDFs, DOCX, XLSX, HTML, and more—without building or maintaining complex ETL pipelines.
```

> Instead of juggling multiple systems and syncing metadata,  Vectorizer handles the entire process: downloading documents from S3, parsing them, chunking text, and generating vector embeddings stored right in Postgres via pgvector. As documents change, embeddings stay up-to-date automatically—keeping your Postgres database the single source of truth for both structured and semantic data.

Reviewer: ```suggestion
Instead of juggling multiple systems and syncing metadata, vectorizer handles the entire process: downloading documents from S3, parsing them, chunking text, and generating vector embeddings stored right in PostgreSQL using pgvector. As documents change, embeddings stay up-to-date automatically—keeping your PostgreSQL database the single source of truth for both structured and semantic data.
```

> ![image](https://github.com/user-attachments/assets/e0145c4b-bc29-4077-8991-fd63d09547cb)

Reviewer: ```suggestion
![create a vectorizer](https://assets.timescale.com/docs/images/console-create-a-vectorizer.png )
```

### Sample dataset for AI testing

> We enable users in the console to import a dataset directly from Hugging Face, based on GitHub docs on pgai. This dataset is ideal for testing vectorizers and can be found in the Import Data page under the Service > Actions tab.

Reviewer: ```suggestion
You can now import a dataset directly from Hugging Face using Timescale Console. This dataset is ideal for testing vectorizers, you find it in the Import Data page under the Service > Actions tab.
```

> ![image](https://github.com/user-attachments/assets/7f12a091-3d71-41a6-8ebf-3ce4b88da869)

Reviewer: ```suggestion
![image](https://assets.timescale.com/docs/images/console-import-huggingface-data.png)
```
